packages/react-recommendations/src/types/RecommendationTransformItems.ts

```diff
@@ -0,0 +1,3 @@
+export type RecommendationTransformItems<TObject> = (
```

Comment:
why is this not called `TransformItems`, as it's not related to recommendations

Reply:
We can inline it so we don't have to name it.

Reply:
Done in https://github.com/algolia/react-recommendations/pull/9/commits/c4e34b585b80325f2f55bef15c2d68451683c7cb